packages/blockchain-api/src/blockscout.ts

```diff
@@ -131,7 +166,7 @@ export class BlockscoutAPI extends RESTDataSource {
           value: new BigNumber(event.value).dividedBy(WEI_PER_GOLD).toNumber(),
           address,
           comment,
-          symbol: CONTRACT_SYMBOL_MAPPING[event.contractAddress.toLowerCase()] || 'unknown',
+          symbol: this.getTokenAtAddress(event.contractAddress) || 'unknown',
```

Comment:
This one you provide a fallback value, but not the for the other uses of `getTokenAtAddress` - might be nice to standardize

Reply:
The fallback above isn't provided as it's for an exchange with our Exchange contract, which must be between gold and dollars. 

The fallback here is for any token transfer, which could be of a new Token whose address we don't recognize